clients/js/test/getAssetsByOwner.test.ts

```diff
@@ -24,4 +24,158 @@ DAS_API_ENDPOINTS.forEach((endpoint) => {
       });
     });
   });
+
+  test(`it can fetch compressed assets by owner with showUnverifiedCollections true (${endpoint.name})`, async (t) => {
+    // Given an owner address.
+    const umi = createUmi(endpoint.url);
+    const owner = publicKey('DASPQfEAVcHp55eFmfstRduMT3dSfoTirFFsMHwUaWaz');
+
+    // When we fetch the asset using the owner with display options.
+    const assets = await umi.rpc.getAssetsByOwner({
+      owner,
+      displayOptions: {
+        showUnverifiedCollections: true,
+      },
+    });
+
+    // Then we expect to find assets.
+    t.true(assets.items.length > 0);
+
+    // And the owner should match.
+    assets.items.forEach((asset) => {
+      t.like(assets.items[0], <DasApiAsset>{
+        ownership: {
+          owner,
+        },
+      });
+    });
+    // And at least one asset should have an unverified collection
+    const assetWithUnverifiedCollection = assets.items.find((asset) =>
+      asset.grouping?.some((group) => group.verified === false)
+    );
+    t.truthy(
+      assetWithUnverifiedCollection,
+      'Expected to find at least one asset with an unverified collection'
+    );
+  });
+
+  test(`it can fetch assets by owner with showCollectionMetadata true (${endpoint.name})`, async (t) => {
+    // Given an owner address.
+    const umi = createUmi(endpoint.url);
+    const owner = publicKey('DASPQfEAVcHp55eFmfstRduMT3dSfoTirFFsMHwUaWaz');
+
+    // When we fetch the asset using the owner with display options.
+    const assets = await umi.rpc.getAssetsByOwner({
+      owner,
+      displayOptions: {
+        showCollectionMetadata: true,
+      },
+    });
+
+    // Then we expect to find assets.
+    t.true(assets.items.length > 0);
+
+    // And the owner should match.
+    assets.items.forEach((asset) => {
+      t.like(assets.items[0], <DasApiAsset>{
+        ownership: {
+          owner,
+        },
+      });
+    });
+
+    // And collection metadata should be present in the grouping for assets that have collections
+    const assetWithCollection = assets.items.find((asset) =>
+      asset.grouping?.some((group) => group.group_key === 'collection')
+    );
+    t.truthy(
+      assetWithCollection,
+      'Expected to find at least one asset with a collection'
+    );
+
+    const collectionGroup = assetWithCollection!.grouping.find(
+      (group) => group.group_key === 'collection'
+    );
+    t.truthy(collectionGroup, 'Expected to find a collection group');
+    t.like(collectionGroup, {
+      group_key: 'collection',
+      group_value: 'Dm1TRVw82roqpfqpzsFxSsWg6a4z3dku6ebVHSHuVo1c',
+      collection_metadata: {
+        name: 'My cNFT Collection',
+        symbol: '',
+        image:
+          'https://gateway.irys.xyz/8da3Er9Q39QRkdNhBNP7w5hDo5ZnydLNxLqe9i6s1Nak',
+        // TODO: Needs to be added again after MTG-1380 is merged
+        //        description: '',
+        //        external_url: ''
```

Comment:
Is this still a TODO?  Can you confirm whether its merged/deployed?

Reply:
This one is blocked by MTG-1493
I’d propose to merge it with that to do and fix it in one of the agreed follow up PRs

Reply:
It's merged to dev, so uncommented it again.